config.yaml

```diff
@@ -72,3 +78,7 @@ options:
     type: string
     description: The username if the SMTP server requires authentication.
     default: ''
+  trusted_key_servers:
+    type: string
+    description: Comma separated list of trusted key servers. Defaults to
```

Comment:
Perhaps describe a bit more what they key servers do here? From reading this I'm not sure what the key servers are and what impact setting them has

Reply:
Sure, thanks